### DISCUSSION TITLE
Mermaid - Github markdown can render diagrams

### DISCUSSION THREAD

**Discussion by krzychu124** [CitiesSkylinesMods/TMPE#1405] [General]
@kvakvs pointed that Github markdown supports now `mermaid` syntax, so we can create graphs, flowcharts, diagrams and more using syntax from https://mermaid-js.github.io/mermaid/#/

example: 
```mermaid
stateDiagram-v2
    [*] --> Still
    Still --> [*]

    Still --> Moving
    Moving --> Still
    Moving --> Crash
    Crash --> [*]
```

More info: https://github.blog/2022-02-14-include-diagrams-markdown-files-mermaid/

**Comment by originalfoo** [comment]
nice!

```mermaid
journey
    title Me reading OP
    section Investigating
        View diagram: 4: Me
        View source: 5: Me
        Try it myself: 7: Me
```
